spec.html

```diff
@@ -30222,11 +30222,7 @@
       <emu-clause id="sec-decimalescape">
         <h1>DecimalEscape</h1>
         <p>The |DecimalEscape| productions evaluate as follows:</p>
-        <emu-grammar>
-          DecimalEscape ::
-            NonZeroDigit
-            NonZeroDigit DecimalDigits
-        </emu-grammar>
+        <emu-grammar>DecimalEscape :: NonZeroDigit DecimalDigits?</emu-grammar>
```

Comment:
Nit: The `?` optional qualifier won't be displayed in the output document (this is a known issue, cf. #536 and #908).